src/components/Fencing/FencingScoreboard.tsx

```diff
@@ -31,8 +36,11 @@ export const FencingScoreboard: React.FC<{gameState: GameState; switchTeams(): v
             <th>Score</th>
             <th>Guesses</th>
           </tr>
-          {_.map(_.values(props.gameState.users), (user, i) => (
-            <tr key={i}>
+          {_.map(props.gameState.users, (user, userId) => (
+            <tr
+              key={userId}
+              className={userId === props.currentUserId ? 'fencing-scoreboard--current-user' : ''}
+            >
               <td>{user.displayName}</td>
               <td>{user.teamId}</td>
```

Comment:
wdyt of moving the switch teams button into a thingy in here?

Reply:
o i like that

Reply:
![image](https://user-images.githubusercontent.com/16993553/109565063-ef71a500-7aaf-11eb-814e-85b681ae230d.png)

any cute icon ideas

Reply:
![image](https://user-images.githubusercontent.com/1988030/109723459-f6a5bb00-7b62-11eb-8a61-8dfaf29c00e8.png)
maybe this